Update experiment-javascript.md (#656)
amplitude/amplitude-docs: content/collections/experiment-sdks/en/experiment-javascript.md

Groups must either be included in the user sent with the fetch request (recommended), or identified with the user via a group identify call from the [Group Identify API](/docs/apis/analytics/group-identify) or via [`setGroup()` from an analytics SDK](/docs/sdks/analytics/browser/browser-sdk-2#user-groups).

```js

> await fetch({groups: {'org name': ['Amplitude']}});

Author: this is incorrect and does not work since the fetch call has to have a user id or device id pass in the user object.

    user_id: 'user@company.com',
    device_id: 'abcdefg',
    user_properties: {
        'premium': true,
    },
    groups: {'org name': ['Amplitude']}
});
```

> If you want to pass in freeform grou properties, you can do something like this:

Reviewer: ```suggestion
To pass freeform group properties, see this example:
```
